BuildServer/Unix/Debian/Resources/DEBIAN/control

```diff
@@ -1,7 +1,7 @@
 Package: mdanse
 Version: 1.0.0
 Architecture: amd64
-Depends: python-setuptools, python-wxmpl, pyro, python-vtk
+Depends: netcdf-bin, libnetcdf-dev, libgtk2.0-0
 Section: Science
 Priority: optional
 Installed-Size: 1000
```

Comment:
Below this, both the 'Maintainer' and 'Description' mention Eric; how should I change these?

Reply:
Please put me as maintainer. 

Keep developed and packaged by Eric as this is for the original version he developed.